products/ubuntu2004/product.yml

```diff
@@ -8,14 +8,15 @@ benchmark_root: "../../linux_os/guide"
 profiles_root: "./profiles"
 
 pkg_manager: "apt_get"
+pkg_manager_config_file: "/etc/apt/apt.conf"
 
 init_system: "systemd"
 oval_feed_url: "https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.focal.cve.oval.xml"
 
 grub2_boot_path: "/boot/grub"
 grub2_uefi_boot_path: "/boot/grub"
 
-aide_bin_path: "/usr/bin/aide.wrapper"
```

Comment:
I believe aide.wrapper still exists in Ubuntu 20.04 and CIS benchmark still explicitly mentions it. I wouldn't change it unless needed. Can you confirm it?

Reply:
It does, and 20.04 "1.3.2 Ensure filesystem integrity is regularly checked" is mentioning `aide.wrapper` unless using timers.
I'll revert that.